Bank/dbo/Tables/Cities.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE Cities
+(
+	Id INT IDENTITY(1,1) NOT NULL,
+	City_Name VARCHAR(25) NOT NULL,

```

Comment:
Read about code style for database naming
If you will have a query with this property, you will have
Cities.City_Name
Better: Cities.CityName
Much better: Cities.Name